.github/workflows/compat.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       matrix:
         # As a general rule, when adding a new version, remove the oldest version
         # Check if the oldest version is EOL or not first.
-        ts: ['4.8', '4.9', '5.0', '5.1', '5.2', '5.3', '5.4', '5.5']
+        ts: ['5.0', '5.1', '5.2', '5.3', '5.4', '5.5']
```

Comment:
Aren't you essentially breaking the rule stated above? Should the list be expanded forwards?

Reply:
It will be! I am keeping the change radius as small as possible with this change because it's already affects a lot of things. Once this is merged, I will follow up with extending the TS version support.